bridge-token/src/lib.rs

```diff
@@ -1,29 +1,33 @@
+use near_contract_standards::fungible_token::metadata::{
+    FungibleTokenMetadata, FungibleTokenMetadataProvider, FT_METADATA_SPEC,
+};
+use near_contract_standards::fungible_token::FungibleToken;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::json_types::{Base64VecU8, ValidAccountId, U128};
+use near_sdk::{
+    assert_one_yocto, env, ext_contract, near_bindgen, AccountId, Balance, PanicOnDefault, Promise,
+    PromiseOrValue, StorageUsage,
+};
+use std::convert::TryInto;
+
+near_sdk::setup_alloc!();
 use admin_controlled::{AdminControlled, Mask};
-use borsh::{self, BorshDeserialize, BorshSerialize};
-use near_sdk::json_types::U128;
-use near_sdk::{env, ext_contract, near_bindgen, AccountId, Balance, Promise};
-
-use near_lib::token::{FungibleToken, Token};
-
-#[global_allocator]
-static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
 const NO_DEPOSIT: Balance = 0;
 
 #[near_bindgen]
-#[derive(BorshDeserialize, BorshSerialize)]
+#[derive(BorshDeserialize, BorshSerialize, PanicOnDefault)]
```

Comment:
Please add `PanicOnDefault` to `BridgeTokenFactory` as well.

Reply:
Done: https://github.com/near/rainbow-token-connector/pull/39/commits/48eeea85f634b01cabf0a1b6aa6d91cc0f16c443